reference/7.1/Microsoft.PowerShell.Utility/Sort-Object.md

```diff
@@ -579,17 +581,23 @@ You can pipe the objects to be sorted to `Sort-Object`.
 ## NOTES
 
 The `Sort-Object` cmdlet sorts objects based on properties specified in the command or the default
-sort properties for the object type. If an object does not have one of the specified properties, the
-property value for that object is interpreted by `Sort-Object` as **Null** and placed at the end of
-the sort order.
+sort properties for the object type. Default sort properties are defined using the
+`DefaultKeyPropertySet` element in a `types.ps1xml` file. For more information, see
+[about_Types.ps1xml](/powershell/module/Microsoft.PowerShell.Core/About/about_Types.ps1xml).
 
+If an object does not have one of the specified properties, the property value for that object is
+interpreted by `Sort-Object` as **Null** and placed at the end of the sort order.
+
+When no sort properties are available, PowerShell attempts to compare the objects themselves.
 `Sort-Object` uses the **Compare** method for each property. If a property does not implement
 **IComparable**, the cmdlet converts the property value to a string and uses the **Compare** method
-for **System.String**. For more information, see [PSObject.CompareTo(Object) Method](/dotnet/api/system.management.automation.psobject.compareto).
+for **System.String**. For more information, see
+[PSObject.CompareTo(Object) Method](/dotnet/api/system.management.automation.psobject.compareto).
 
 If you sort on an enumerated property such as **Status**, `Sort-Object` sorts by the enumeration
-values. **Stopped** has a value of **1** and **Running** has a value of **4**. **Stopped** is sorted
-before **Running** because of the enumerated values. For more information, see [ServiceControllerStatus](/dotnet/api/system.serviceprocess.servicecontrollerstatus).
+values. For Windows services, **Stopped** has a value of **1** and **Running** has a value of **4**.
```

Comment:
Are we supposed to say **Windows** rather than **Microsoft Windows**?

Reply:
**Windows** is sufficient here.